data/json/bionics.json

```diff
@@ -364,15 +358,15 @@
     "type": "bionic",
     "name": { "str": "Directional EMP" },
     "description": "Surgically mounted in the palm of your hand are small parabolic EMP field generators.  You may use power to fire a wide, but short-ranged blast which will disable electronics and robots.",
-    "occupied_bodyparts": [ [ "arm_r", 1 ], [ "hand_r", 2 ] ],
+    "occupied_bodyparts": [ [ "arm_l", 6 ], [ "hand_l", 1 ] ],
     "act_cost": "50 kJ"
   },
   {
     "type": "bionic",
     "id": "bio_emp_armgun",
     "name": { "str": "EMP Projector" },
     "description": "A ranged EMP generator system is implanted on the palm of your right hand and arm.  The system fires precise, single-target pulses that require time to aim.  Extremely effective against electronic targets but mostly useless otherwise.",
-    "occupied_bodyparts": [ [ "arm_r", 10 ], [ "hand_r", 5 ] ],
+    "occupied_bodyparts": [ [ "arm_r", 8 ], [ "hand_r", 2 ] ],
     "act_cost": "50 kJ",
```

Comment:
Should this really take the same energy as a wide blast?

Reply:
Long-range electric attack, the air is a good insulator.
The power value itself probably comes from some PR long ago which audited bionics' power consumption based on irl values.